app/src/components/Forms/Info/DetailedInfo.js

```diff
@@ -0,0 +1,247 @@
+              As candidates collect small contributions from Portlanders, the
+              City matches them 6-to-1 on the first $50. So if you give a
+              candidate $10 a month, the City will match that with $60 each time
+              until you’ve given $50 and the City has provided the candidate
+              $300 in matching funds for you.
+            </li>
+            <li>
+              You can get matched for each race, so you could give to someone
+              running for Mayor and get matched, and to candidates in each of
+              the Commission races and get matched, and to a candidate in the
+              Auditor’s race and get matched.
+            </li>
+          </ul>
+          <h3>What it's for</h3>
+          <p>
+            Our government works best when every person is engaged in the
+            elections process and can make a difference in the community. The
+            program fosters an inclusive democratic process where everyone
+            participates and everyone’s contributions matter. A community where
+            people from all walks of life can run for and win office, while
+            avoiding concerns about the influence of large donations in City
+            elections.
+          </p>
+          <p>
+            We’re strongest when our elected city leaders reflect the full range
+            of talent and lived experience that Portland has to offer, and when
+            the community trusts that elected leaders share the community’s
+            values. People from every neighborhood in Portland should have
+            meaningful opportunities to influence who is elected to City
+            offices, and to run effective citywide campaigns.
+          </p>
+
+          <h3>About the software</h3>
+          <p>
+            This dashboard and the software that powers it was built in
+            partnership between the City’s Open and Accountable Elections
+            program and the{' '}
+            <a
+              href="https://civicsoftwarefoundation.org"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Civic Software Foundation
+            </a>
+            .
+          </p>
+
+          <p>
+            The software is{' '}
+            <a
+              href="https://github.com/hackoregon/openelections/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              open-source
+            </a>{' '}
+            which means that anyone can scrutinize the code — which is essential
+            for true transparency and accountability. It also means that other
+            governments interested in visualizing their campaign finance data
+            can use some or all of the open-source code.
+          </p>
+
+          <p>
+            The software was built by an interdisciplinary team of volunteers
+            and contractors working together through{' '}
```

Comment:
It's unclear what Hack Oregon's relationship is to CIVIC. Maybe say something like "...working together through the Civic Software Foundation's Hack Oregon program."

Reply:
Revised to your suggestion